documentation/projects/proposals/rekognition_data/20240530-implementation_plan_augment_catalog_with_rekognition_tags.md

```diff
@@ -506,6 +513,28 @@ For local testing, a small sample of the Rekognition data could be made
 available in the local S3 server
 [similar to the iNaturalist sample data](https://github.com/WordPress/openverse/blob/82282a00abdaed21e8381052a874d8ab9a4f7e0a/catalog/compose.yml#L98-L101).
 
+### Filter Rekognition tags
+
+Once the new Rekognition tags have been inserted into the catalog, we will want
+to remove the blanket-filter for all Rekognition tags that was put in place in
+the [preemptive filtering step](#preemptively-filter-rekognition-tags). We will
+also need to add the logic for filtering out the tags that were determined
+should be excluded in the
+[determine excluded labels step](#determine-excluded-labels).
+
+For all machine-generated labels, we will employ an inclusion-based filtering
+process. This means that we will only filter out labels that match the list of
+approved labels, which prevents labels that are unreviewed from appearing in the
+downstream dataset. This can be added to the `alter_data` step of the data
+refresh (see #4684) and would only be applied to tags where the `provider` was
+not the record's `provider`.
+
+We will also add a step for recording if a label was not in the inclusion list
+_and_ if it did not exist in a full list of all reviewed labels from the
+provider. These "unreviewed" labels can be surfaced as part of the data refresh,
+so maintainers can review them and decide if they should be included in the
+inclusion list.
```

Comment:
Just a nit about can/should/must etc. "can" reads very much as an "option" to me, rather than a certain "will be".

```suggestion
provider. These "unreviewed" labels should be surfaced as part of the data refresh,
so maintainers can review them and decide if they should be included in the
inclusion list.
```